pkg/k8sutil/cluster.go

```diff
@@ -1,70 +1,57 @@
 package k8sutil
 
 import (
+	"context"
+	"fmt"
+	v1 "k8s.io/api/core/v1"
 	"time"
 
 	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+	bootstrapapi "k8s.io/cluster-bootstrap/token/api"
 	bootstraputil "k8s.io/cluster-bootstrap/token/util"
-	bootstraptokenv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/bootstraptoken/v1"
-	kubeadmapiv1beta3 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"
-	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
-	tokenphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/bootstraptoken/node"
-	"k8s.io/kubernetes/cmd/kubeadm/app/phases/copycerts"
-	kubeadmconfig "k8s.io/kubernetes/cmd/kubeadm/app/util/config"
 )
 
 // GenerateBootstrapToken will generate a node join token for kubeadm.
 // ttl defines the time to live for this token.
 func GenerateBootstrapToken(client kubernetes.Interface, ttl time.Duration) (string, error) {
 	token, err := bootstraputil.GenerateBootstrapToken()
 	if err != nil {
-		return "", errors.Wrap(err, "generate kubeadm token")
+		return "", errors.Wrap(err, "failed to generate bootstrap token")
```

Comment:
why renaming this text? will this impact error messages that exist in our docs, community, analyzers, etc?  Unless there's a product reason, feels unecessary and risky

Reply:
I can revert this.